### DISCUSSION TITLE
v1.7.0

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#480] [Announcements]
## What's new
- Updated for macOS Sequoia.
- The integrated viewer now provides syntax highlighting using the Lexilla library.
- The integrated viewer no longer uses toolbars; instead, a status bar is used to save space.
- The user guide has been significantly expanded.
- Updated the application icon.
- Added an option for natural collation when sorting filenames.
- Fixed a bug causing unstable ordering of tool icons on the toolbar.
- The "Go To" popup now allows numeric keys to filter items.
- Added a new hotkey: Ctrl+Return to display a context menu.
- Fixed an issue with incomplete FTP transfers.
- Fixed an issue where an overlapped terminal caused the mouse cursor to display the wrong type.
- Fixed a broken Shift+Tilde shortcut.
- Removed the built-in checksum calculation feature.
- Cmd+B now adds the current directory of a panel instead of the path to the currently focused item.
- Fixed an issue where excessive refresh requests during file operations caused incorrect behavior in file panels.
- Numerous small UI polish improvements have been made.

## New Contributors
* @fabsantana made their first contribution in https://github.com/mikekazakov/nimble-commander/pull/428

**Full Changelog**: https://github.com/mikekazakov/nimble-commander/compare/v1.6.0...v1.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mikekazakov/nimble-commander/releases/tag/v1.7.0'>v1.7.0</a>.</em>

**Comment by evmin** [comment]
Thank you - was a nice present this holiday period.
